app/src/main/java/org/simple/clinic/bloodsugar/BloodSugarReading.kt

```diff
@@ -0,0 +1,3 @@
+package org.simple.clinic.bloodsugar
+
+data class BloodSugarReading(val value: Int, val type: BloodSugarMeasurementType)
```

Comment:
I am bringing the reading value and type together to give them meaning at a domain level. This lets us avoid [Primitive Obsession](https://enterprisecraftsmanship.com/posts/functional-c-primitive-obsession/) and lets us avoid unnecessary creation of validator classes like we currently do for `BloodPressureMeasurement`.

With this approach, this class can:
- Be responsible for its own validation.
- Be instantiated and passed around independently and expose intent through its type rather than the primitive parameter names.

Reply:
Nicely done :)